src/rider/main/kotlin/com/jetbrains/rider/plugins/unreal/toolWindow/UnrealToolWindowFactory.kt

```diff
@@ -77,14 +72,15 @@ class UnrealToolWindowFactory(val project: Project)
                 }
             }
             filter()
-        })
+        }
 
-        UnrealPane.verbosityFilterActionGroup.addItemListener({
+        UnrealPane.verbosityFilterActionGroup.addItemListener {
             filter()
-        })
+        }
 
         UnrealPane.timestampCheckBox.addChangeListener { event ->
-            filter()
+            val showTimestamp = UnrealPane.timestampCheckBox.isSelected
+            UnrealPane.currentTimeConsoleView.isVisible = showTimestamp
```

Comment:
There's no such property in changeEvent. There are also 2 other listeners but they don't seem relevant.
In our existing code in intellij I also see that we call isSelected() for the checkbox instead of extracting the state from the event.